### ISSUE TITLE
Add 'Layer Normalization' and 'Batch Normalization' layers

### ISSUE THREAD

**Issue by MateuszSzymanski2912** [MateuszSzymanski2912/PikminFlow#3]
Layer and batch normalization are techniques used to improve and stabilize learning process. They can help with the vanishing gradient problem that appear especially in deep neural networks. From implementation point of view, both layers do the same, but they do it on different axis. These layers dont have hyperparameters, so user doesn't have to input anything.
